### DISCUSSION TITLE
Opción a deshabilitar la escritura predictiva

### DISCUSSION THREAD

**Discussion by ghost** [gusmanb/ZXBasicStudio#43] [Ideas]
Creo que sería bueno tener la posibilidad de habilitar/deshabilitar la escritura predictiva del editor de texto. A veces, según lo que se escriba, la máquina te coloca cadenas de texto sin avisar, y tienes que ir vigilando el nombre de variables. Por ejemplo, para una variable "y", me pone siempre "inkeys", y tengo que ir rectificando cada dos por tres.
Además, a veces te lo coloca en mayúsculas, y a veces en minúsculas, en función de la posición de la línea donde te encuentres.

**Comment by SirRickster** [comment]
Creo que fué @Duefectu el que propuso que se pudiera seleccionar un mínimo de dos o tres letras para que empieze a mostrar sugerencias o el autocompletado.

**Comment by gusmanb** [comment]
En cuanto vuelva le meto mano (llevo dos semanas con un puñetero constipado de narices y fiebre) que tengo claro lo que hacer, poner una opción para que solo se active con ctrl+space y otra que especifique la longitud mínima del texto para que se active si está en modo automático.

**Reply by Duefectu** [reply]
Cuídate!!!
